README.md

```diff
@@ -8,147 +8,244 @@
 
 ##Install
 
-Requires jquery and bootstrap
 
 `npm install react-maxwell-modal --save`
 
 
 ##Usage
 
-There are three types of modals available
+There are four types of modals available
 
-### Modal
+### Content Modal
 
-This is the basic modal. It has two types of configurations
-
-* Content
-* Header, Body, Footer
-```javascript
-
-$('body').append('<div id="modal-container"></div>');
-  React.render(<Modal
-      body="foo"
-      title="foo"
-      show={true}
-      onYes={function(){console.log('onYes'); return true;}}
-      onShow={function(){console.log('onLoad');}}
-      onHide={function(){console.log('onHide');}} />, document.getElementById("modal-container"));
-```
 
 ####Props
 
 
 
 |Prop|Type| Required|
 |-----|----|--------|
-|footer|any| false|
-|header|any| false|
-|content|any| true|
-|body|any| false|
 |onShow|function| false|
 |onHide|function| false|
-|title|string| false|
-|dismissable|bool| false|
-|show|bool| false|
-
-####Content
+|isOpen|bool| false|
+|shouldCloseOnOverlayClick|bool| false|
 
-If the content property is present it will operate in content mode.
 
 ```javascript
-$('body').append('<div id="modal-container"></div>');
-React.render(<Modal
-    content="foo"
+import { ContentModal } from 'react-maxwell-modal';
+
+class ContentModalExample extends React.Component {
+  constructor(props) {
```

Comment:
can always import the Component by itself and extend that (as opposed to React.Component)